cql3/statements/alter_table_statement.cc

```diff
@@ -32,19 +32,25 @@ namespace statements {
 
 static logging::logger mylogger("alter_table");
 
-alter_table_statement::alter_table_statement(cf_name name,
+alter_table_statement::alter_table_statement(uint32_t bound_terms,
+                                             cf_name name,
                                              type t,
                                              std::vector<column_change> column_changes,
                                              std::optional<cf_prop_defs> properties,
                                              renames_type renames)
     : schema_altering_statement(std::move(name))
+    , _bound_terms(bound_terms)
     , _type(t)
     , _column_changes(std::move(column_changes))
     , _properties(std::move(properties))
     , _renames(std::move(renames))
 {
 }
 
+uint32_t alter_table_statement::get_bound_terms() const {
+    return _bound_terms;
+}
+
```

Comment:
I feel like this boilerplate should be handled automatically, but that's of course out of scope here.

Reply:
Yes, and moreover, the name "bound_terms" is really confusing (it sounds like it might be a vector, but in fact it's an integer - what does "3" mean?). But I couldn't complain because all the other classes already have the same thing :-(